<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>620</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>FileRenamer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Rename settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="FormFormatSettings" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>274</width>
             <height>500</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRename"/>
   <addaction name="actionPause"/>
   <addaction name="actionCancel"/>
   <addaction name="actionRedo"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/forward.png</normaloff>:/res/images/forward.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Start renaming</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/pause.png</normaloff>:/res/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause/Resume</string>
   </property>
   <property name="toolTip">
    <string>Pause/Resume</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/stop.png</normaloff>:/res/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="toolTip">
    <string>Cancel renaming</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/backward.png</normaloff>:/res/images/backward.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo renaming</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/res/images/exit2.png</normaloff>:/res/images/exit2.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormFormatSettings</class>
   <extends>QWidget</extends>
   <header>widgets/FormFormatSettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
